### DISCUSSION TITLE
Timeline Calendar

### DISCUSSION THREAD

**Discussion by bahadirk5** [shadcn-ui/ui#4999] [Show and tell]
### Description

A timeline calendar for reservations.

github repo: https://github.com/bahadirk5/timeline-calendar

<img width="1348" alt="timeline-calendar" src="https://github.com/user-attachments/assets/5e0926f3-3e13-45c2-bffb-c378d5333d9f">

**Comment by blibbers** [comment]
Yo is this still available? It's currently 404ing

**Reply by bahadirk5** [reply]
Fixed
